### PR TITLE
Add 4.x branch for doctrineORMAdmin

### PR DESCRIPTION
**Pull request by VincentLanglet**

### CODE REVIEW

**Comment by VincentLanglet** [comment]
POC https://github.com/sonata-project/SonataDoctrineORMAdminBundle/pull/1488

File: config/projects.yaml [hunk 1]
```diff
@@ -247,8 +247,14 @@ doctrine-orm-admin-bundle:
...
             variants:
-                symfony/symfony: ['4.4']
+                symfony/symfony: ['4.4', '5.3']
                 sonata-project/admin-bundle: ['dev-master']
+        4.x:
+            php: ['7.3', '7.4', '8.0']
+            php_extensions: ['zip']
+            variants:
+                symfony/symfony: ['4.4', '5.3']
+                sonata-project/admin-bundle: ['^4@rc']
```
Comment (jordisala1991) [inline config/projects.yaml:257]: why don't you use directly 4?

Comment (VincentLanglet) [reply config/projects.yaml:257]: Indeed the current check is 
```
'dev-master' === $this->version ? $this->version : ($this->version.'.*')
```
so it won't work like this.

Comment (VincentLanglet) [reply config/projects.yaml:257]: But 4.* doesn't work https://github.com/sonata-project/SonataDoctrineORMAdminBundle/pull/1488

File: config/projects.yaml [hunk 2]
```diff
@@ -247,14 +247,20 @@ doctrine-orm-admin-bundle:
             php: ['7.3', '7.4', '8.0']
             php_extensions: ['zip']
             variants:
-                symfony/symfony: ['4.4']
+                symfony/symfony: ['4.4.*', '5.3.*']
                 sonata-project/admin-bundle: ['dev-master']
```
Comment (jordisala1991) [inline config/projects.yaml:251]: this one will start failing. We will need to reconfigure the aliases of master to 5.x right? and add dependency of each other.

**Review by jordisala1991 — APPROVED** [review]

File: config/projects.yaml (hunk 2 above)
Comment (jordisala1991) [reply config/projects.yaml:251]: Nvm ignore this.

**Review by jordisala1991 — APPROVED** [review]

**Pull request merged**